pkgs/by-name/in/insync-nautilus/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  dpkg,
+  gnome,
+  insync
+}:
+
+stdenv.mkDerivation rec {
+  pname = "insync-nautilus";
+  version = lib.getVersion insync;
+  pyproject = true;
+
+  # Download latest from: https://www.insynchq.com/downloads/linux#nautilus
+
+  src = fetchurl {
+    url = "https://cdn.insynchq.com/builds/linux/insync-nautilus_${version}_all.deb";
+    sha256 = "sha256-aB1/ZzcQH3T1lviMZO8jXbtdbe4TW20f0TAcv4HDOGI=";
+  };
+
+  nativeBuildInputs = [
+    dpkg
+  ];
+
+  buildInputs = [
+    gnome.nautilus-python
+    insync
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -R usr/share $out/
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
Might want to set `sourceProvenance` here.

Reply:
okay, I set fromSource

Reply:
done